packages/uniforms/src/JSONSchemaBridge.js

```diff
@@ -26,6 +26,7 @@ export default class JSONSchemaBridge extends Bridge {
 
     this.schema = schema;
     this._compiledSchema = {};
+    this._rootSchema = this._getRootSchema();
```

Comment:
I don't like this idea. How about resolving within `this.schema` it to actually keep only that one field? Like, if it's an object: done; if it's a `$ref` object, then construct a new object: `{...schema, ...resolveRef(schema.$ref, schema)}`. It'll work, right?

Reply:
My intention was to keep schema as is, but i can think about other solution